SpringProject/src/main/resources/db.migration/V1__createSchema.sql

```diff
@@ -0,0 +1,8 @@
+create schema conferenceFinder;
+create table conferenceFinder.conference (
+	id long,
```

Comment:
Wrong indentation here, so this line should be at the same level as the lines below.